CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * Support pausing reconciliation of KafkaTopic CR with annotation `strimzi.io/pause-reconciliation`
 * Update cruise control to 2.5.55
 * Support for broker load information added to the rebalance optimization proposal. Information on the load difference, before and after a rebalance is stored in a ConfigMap
+* Added support for `controller_mutation_rate quota`. Creation/Deletion of topics and creation of partitions can be configured through this .
```

Comment:
```suggestion
* Added support for `controller_mutation_rate` quota. Creation/Deletion of topics and creation of partitions can be configured through this.
```

Reply:
I applied the above the change and rebased the PR